source/_integrations/fluss.markdown

```diff
@@ -0,0 +1,51 @@
+---
+title: Fluss+
+description: Instructions on how to integrate Fluss+ devices within Home Assistant.
+ha_category:
+  - Button
+ha_release: "2024.11"
+ha_iot_class: Cloud Polling
+ha_codeowners:
+    -'@fluss'
+ha_domain: fluss
+ha_config_flow: true
+ha_platforms:
+  - button
+ha_integration_type: integration
+related:
+  - docs: /docs/configuration/
+    title: Configuration file
+---
+
+The **Fluss+** {% term integration %} allows you to control [Fluss+](https://fluss.io/) devices through Home Assistant.
+
+## Prerequisites
+
+- A Fluss+ account
+- The Fluss+ app installed on your smart phone.
+- Fluss+ device connected to your Wi-Fi network and has internet access.
+- During setup of the integration in Home Assistant, you will be prompted to input your API key.
+   - You can request an API key in the Fluss+ app under your profile settings.
+
+{% include integrations/config_flow.md %}
+
+## Setup
+To add Fluss+ to your installation, go to **Settings** -> **Devices & Services** in the UI, click the button with `+` sign and from the list of integrations select **Fluss+**.
+You will be prompted to enter your API key.
+Once the integration is set up, you will see a list of your Fluss+ devices.
```

Comment:
```suggestion
```
this is more or less a duplicate of what is provided by the include element above.

![image](https://github.com/user-attachments/assets/ee6f6ae1-0f62-4ea5-a4ce-40f0c4bb9563)